src/memoize/signature.ts

```diff
@@ -13,7 +13,7 @@
  * We must hand-write a signature for each custom class we wish to support.
  */
 
-import * as isPlainObject from "is-plain-object";
+import isPlainObject from "is-plain-object";
```

Comment:
alternatively, could've kept the `import * as isPlainObject` and replaced `isPlainObject(obj)` with `isPlainObject.default(obj)`, but this seemed cleaner